.github/workflows/deploy.yml

```
@@ -47,20 +47,22 @@ jobs:
 
       # https://github.com/pypa/cibuildwheel
       - name: Build wheels
-        uses: pypa/cibuildwheel@v3.1.4
+        uses: pypa/cibuildwheel@v3.2.0
```

Do we need to pin to x.y.z or shall we use x.y? (cibuildwheel doesn't do plain x) 
```suggestion
        uses: pypa/cibuildwheel@v3.2
```